wikdict-query/wikdict_query/languages.json

```diff
@@ -1,1068 +1,1068 @@
 [
   {
-    "lang": "http://www.wikidata.org/entity/Q9035",
+    "lang": "https://www.wikidata.org/entity/Q9035",
```

Comment:
These are identifiers and only incidentally URLs (you can use them in SPARSQL queries), so let's not change them.

Reply:
Alright, let me revert that part

Reply:
Done, pushed